cli/src/main/java/com/liferay/blade/cli/BladeCLI.java

```diff
@@ -303,6 +304,18 @@ public void run(String[] args) throws Exception {
 
 				_commandArgs.setBase(baseDir);
 
+				if (_commandArgs instanceof CreateArgs) {
```

Comment:
We don't want to to make this class have to know about CreateArgs, it should be agnostic.

Reply:
So this code should live in the CreateCommand class instead.